src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonConverterAttribute.cs

```diff
@@ -21,7 +23,7 @@ public class JsonConverterAttribute : JsonAttribute
         /// Initializes a new instance of <see cref="JsonConverterAttribute"/> with the specified converter type.
         /// </summary>
         /// <param name="converterType">The type of the converter.</param>
-        public JsonConverterAttribute(Type converterType)
+        public JsonConverterAttribute([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] Type converterType)
```

Comment:
These don't seem to have corresponding entries in the `ref` file. Or am I missing it?

Reply:
Thanks. I've added them now.